store/sqlstore/utils.go

```diff
@@ -18,6 +18,10 @@ import (
 	"github.com/go-sql-driver/mysql"
 )
 
+const (
+	paginationLimit = 1000
```

Comment:
It's just that when I looked at a few hard-coded limits, all of them had the 1000 limit.
But yeah, since it's not an established std. I'll move it to the local PN store.